packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -118,7 +118,7 @@ contract SystemConfig is OwnableUpgradeable, Semver {
                 systemTxMaxGas: 0,
                 maximumBaseFee: 0
             }),
-            _startBlock: 0,
+            _startBlock: 1,
```

Comment:
maybe something like `MAX_UINT` would be a better placeholder. Since the system will eventually reach block 1?

Reply:
Thats a great idea